ProcessMaker/processmaker
# Feature/FOUR-16114: Re-run wizard button

## How to test
- Go to Processes
- Create a Process from wizard Templates
- Open process with wizard template
- Check the proces Info Header, a Link with "Re-run Wizard" must be on the header.

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-16114

## Code Review Checklist

ci:next
